core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -267,7 +267,7 @@ public FileInfo getFileInfo(String ufsFullPath, GetStatusPOptions options) throw
     DoraMeta.FileStatus status = mUfsStatusCache.getIfPresent(ufsFullPath);
     if (syncIntervalMs >= 0 && status != null) {
       // Check if the metadata is still valid.
-      if (System.currentTimeMillis() - status.getTs() > syncIntervalMs) {
+      if (System.nanoTime() - status.getTs() > syncIntervalMs * Constants.MS_NANO) {
```

Comment:
why do we need nano time? nano time is more for performance measurement purpose or elaspsed time intervals, and it's not related to current date/time. if the status record some time from previous persistent metastore or from UFS, this may not be good to use.

and do we need that much precision? If system.currentTimeMillis can fulfill the need, then nano time may not be needed?

Reply:
Suppose we have the following case:
T1. GetStatus() from client to worker.
T2. Modify the file metadata directly from client to UFS.
T3. GetStatus() from client to worker again. This time, syncIntervalMs is set to ZERO. It means the client wants the metadata to be refreshed. 
     But you see, the cached Status has a timestamp of T1, and the current timestamp is T3.
     If we use milliseconds as the timestamp, T3 might be equal to T1, if the client and worker have very good performance. This will cause the metadata not to be refreshed.

Nano is much more precise in this case. We are not using nano time to compare it with the timestamp from UFS.
This nano time is used as the timestamp of cache item, and only be used by this particular worker to compare the cached item.